en/release_note/release_note_latest_ver.rst

```diff
@@ -26,56 +28,69 @@ For CUBRID 9.3, please find https://www.cubrid.org/manual/en/9.3.0/release_note/
 Overview
 ========
 
-CUBRID 11.2 is the latest stable version that includes new features, significant changes and enhancements.
+CUBRID 11.3 is the latest stable version that includes new features, significant changes and enhancements.
 
 .. TODO: UPDATE WITH DETAILS.
 
-CUBRID 11.2
+CUBRID 11.3
 
-* is a version with improved connectivity.
+* improves connectivity by improving database links.
```

Comment:
How about changing as the following ?
improves connectivity by enhancing database links.

Reply:
Reflected it